tools/packaging/kernel/build-kernel.sh

```diff
@@ -538,6 +538,16 @@ main() {
 
 	[ -z "${subcmd}" ] && usage 1
 
+	if [[ ${build_type} == "experimental" ]] && [[ ${hypervisor_target} == "dragonball" ]]; then
+		build_type="dragonball-experimental"
+		if [ -n "$kernel_version" ];  then
+			kernel_ver=$(get_major_kernel_version "${version}")
```

Comment:
```suggestion
			kernel_ver=$(get_major_kernel_version "${kernel_version}")
```

Reply:
Then here you declare `kernel_ver` but does not actually use it, which makes me thing that ....

Reply:
good catch!!! I've changed kernel_ver to kernel_major_version and also if [[ ${kernel_version} != "5.10" ]]; then to if [[ ${kernel_major_version} != "5.10" ]]; then